charts/rclone-copy/templates/rclone-cron.yaml

```diff
@@ -52,6 +52,9 @@ spec:
             args:
               - copy
               - -v
+            {{-  range .Values.arguments }}
```

Comment:
If you change this, then please also add a corresponding field to the values. Probably an empty array but you should provide an example with comments. Or remove this if it is not needed

Reply:
It's not needed this time. But should we keep it for the future, since allowing arguments is useful?

Reply:
If it is something useful then feel free to add it but also update the values (and don't forget to package :-) )